app/assets/stylesheets/_layout.scss

```diff
@@ -8,6 +8,15 @@ body {
   padding-left: 0;
   padding-right: 0;
   height: 100%;
+
+  .flash-message-row {
```

Comment:
This one is intended to modify the standard  `.flash-message-row` style on pages using `.container-fluid` instead of `.container`